src/sentry/tasks/unmerge.py

```diff
@@ -216,8 +216,6 @@ def migrate_events(
 
     event_id_set = set(event.id for event in events)
 
-    Event.objects.filter(project_id=project.id, id__in=event_id_set).update(group_id=destination_id)
-
     for event in events:
         event.group = destination
 
```

Comment:
Is the call to tagstore.update_group_for_events right below supposed to work with event_id ?

Reply:
Thanks, you're right it won't work. I've updated this to look up the correct IDs if we are using legacy tag storage. Once we get rid of this backend we can delete this code.